Fixed: Link Party feature stop the EECA rules globally(OFBIZ-12424)

rohitkou opened the pull request:
Fixed: Link Party feature stop the EECA rules globally(OFBIZ-12424)

Create and use the default-no-eca delegator instead of setting EntityEcaHandler object null on delegator.

JacquesLeRoux reviewed (approved):
Thanks Rohit,

That sounds indeed like the right thing to do to me

mbrohl commented:
@rohitkou can you please remove the merge commit from the PR and possibly do a rebase instead?
Thanks!

JacquesLeRoux commented:
Else, as it's a very simple change we can also do it manually and backport...

rohitkou commented:
Thanks you @JacquesLeRoux and @mbrohl, I've extracted the changes into a patch file and added it to ticket OFBIZ-12424. We can close this pull request now.
